src/renderers/shaders/ShaderChunk/begin_vertex.glsl

```diff
@@ -1,2 +1,19 @@
 
+#ifndef INSTANCE_TRANSFORM
+
 vec3 transformed = vec3( position );
+
+#else
+
+#ifndef INSTANCE_MATRIX 
+
+	mat4 _instanceMatrix = getInstanceMatrix();
+
+	#define INSTANCE_MATRIX
```

Comment:
could not find a more elegant solution, it's scattered around in different shaders, and there is a race condition